modules/custom/d_p_node/templates/node--blog-post--d-small-box.html.twig

```diff
@@ -61,5 +61,5 @@
 #}
 
 {% include "@d_p_node/d-small-box.html.twig" with {
-  image: content.field_blog_main_image.0,
+  image: content.field_blog_media_main_image.0,
```

Comment:
a jak nie będzie 0 to co wtedy się stanie?

Reply:
nic, nie bedzie obrazka :)